external/common_functions.sh

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# All supported JVMs
+all_jvms="hotspot openj9"
+
+# All supported packges
+all_packages="jdk jre"
+
+# All supported distros
+all_os="alpine debian debianslim ubi ubi-minimal centos clefos ubuntu windowsservercore-ltsc2016"
+
+# All support versions
+all_versions="8 9 10 11 12 13"
+
+# All supported builds
+all_builds="slim full"
+
+# All supported tests
+all_tests="derby elasticsearch jenkins kafka lucene-solr openliberty payara quarkus scala thorntail tomcat tomee wildfly wycheproof"
+
+function check_version() {
+	version=$1
+	case ${version} in
+	8|9|10|11|12|13)
```

Comment:
I would love to do that, but I am not sure how do that in bash. Dynamically have a case statement with options from an array. I guess I could just make a for loop instead of a case statement.